harg.cabal

```diff
@@ -67,9 +67,9 @@ library
     ghc-options:        -Wall -Wno-unticked-promoted-constructors
     build-depends:
         base >=4.7 && <5,
-        aeson >=1.4.2 && <1.5,
+        aeson >=2.0 && <2.1,
```

Comment:
Stackage LTS's that support GHC 9 also require `aeson` 2.
I'm unfamiliar with how to support multiple library versions, but I _think_ it's possible with CPP macros to support both versions of `aeson` (if necessary).